open_discussions/settings.py

```diff
@@ -407,6 +407,10 @@
     'JWT_AUTH_HEADER_PREFIX': 'Bearer',
 }
 
+EXTERNAL_LOGIN_URL = get_string('OPEN_DISCUSSIONS_EXTERNAL_LOGIN_URL', None)
```

Comment:
would it make sense to call this `MM_EXTERNAL_LOGIN_URL`? I'm just thinking that if, at some point, users can authenticate through another service we may have multiple external login URLs, so it might make sense to go ahead and name this one specifically now and thereby avoid changing the name in the future.

Reply:
Sure